### DISCUSSION TITLE
Self host vs Get Hosted

### DISCUSSION THREAD

**Discussion by Demo-TCO** [KelvinTegelaar/CIPP#1245] [General]
Hi,

I'm wanting to understand a few items regarding Azure vs Hosted version. 
I am already using Azure to host CIPP and have been running into a few errors which has made me consider the hosted version available from the website. A few questions I would like answered if anyone is able to help out.
What level of access would the host have to the tenant/product?
How would new users be assigned from our side? 
Where will the data be hosted? We're based in Australia and due to some requirements, we'd like to ensure data is hosted in AU.
What sort of data is collected?

Hopefully someone is able to help answer this to try and understand the difference between the 2. Hoping that the Sponsor/hosted version allows for better connectivity and speeds for us.

**Comment by KelvinTegelaar** [comment]
Good questions, let me try to work them off in a list format so I don't miss any:

- What level of access who the host have
   - We will have access to the resource group, the storage account, and all Azure sided resources, but by default we do not have access to your keys, tenant information, etc. To make sure you can check our level of access and all of our actions within Azure, which has an immutable audit log. You'll be able to access this at any time, and connect it to a SIEM/SOC if you have one.

- How would new users be assigned
   - you'll be able to email the helpdesk, or use your own backend access to do this yourself.

- Where will data be hosted
     - Any location you choose, during hosted setup you choose your own location and nearest Azure DC etc.
 
- What sort of data is collected
   - Nothing. We do not collect any information for any analysis purposes what so ever. We do monitor application usage to see if the application(Function app, Static web app) is healthy so we can monitor that.

If you're having speed issues on your local version, make sure you've enabled Run From Package mode. If you forget that there is a pretty big error on your dashboard pointing you how to set that up. Hope that answers your questions!

**Reply by Demo-TCO** [reply]
Hey Kelvin,

Thank you so much for the quick reply. Apologies for the delayed reply from my side as well. I have enabled Run From Package Mode and it has helped with the performance, but I think due to it being a test tenant, I am receiving API errors.

I'm just trying to compare from my end which way to take. Azure Host or Paid Hosting and what the benefits are of paid hosting. Hopefully this isn't too broad a question, just trying to understand the differences so I can make a decision for my organisation and deploy asap.

Thanks again!
